src/config.js

```diff
@@ -1,3 +1,7 @@
-const API = 'http://10.58.7.23:8000';
+const API = {
+  login: 'http://10.58.7.23:8000',
+  signup: 'http://10.58.7.23:8000',
+  shop: 'http://10.58.3.57:8000',
+};
```

Comment:
객체로 만들어 사용하는 부분 좋은 것 같은데 같은 ip를 서로 다른 프로퍼티로 관리할 필요가 있을까 하는 생각이 듭니다.
차라리 / 이후의 엔드포인트들도 객체 안에 넣어주는게 어떨까요?
```suggestion
const BASE_URL = 'http://10.58.7.23:8000'
const API = {
  login: `${BASE_URL}/users/login`,
  signup: `${BASE_URL}/users/signup`,
  shop: `${BASE_URL}/shop`,
};
```

Reply:
지금 요청하는 아이피 주소가 다 다르게 나와서 하나로 관리를 못하고 있습니다.  ㅜㅜ